### PR TITLE
Add french traduction interface.

### PR DESCRIPTION
**Pull request by KevinJaniky**
Add fr directory to translate this interface to French .

### CODE REVIEW

File: resources/lang/fr/calendar.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+        3 => 'Mars',
+        4 => 'Avril',
+        5 => 'Mai',
+        6 => 'Juin',
+        7 => 'Juillet',
+        8 => 'Aout',
+        9 => 'Septembre',
+        10 => 'Octobre',
+        11 => 'Novembre',
+        12 => 'Decembre'
```
Comment (samnela) [inline resources/lang/fr/calendar.php]: Décembre

File: resources/lang/fr/calendar.php [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<?php
+
+return [
+    'months' => [
+        1 => 'Janvier',
+        2 => 'Fevrier',
```
Comment (samnela) [inline resources/lang/fr/calendar.php]: Février

File: resources/lang/fr/calendar.php [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+        3 => 'Mars',
+        4 => 'Avril',
+        5 => 'Mai',
+        6 => 'Juin',
+        7 => 'Juillet',
+        8 => 'Aout',
```
Comment (samnela) [inline resources/lang/fr/calendar.php]: Août

File: resources/lang/fr/fields.php [hunk 4]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+return [
+    'name' => 'Nom',
+
+    'avatar' => 'Avatar',
+    'email' => 'E-mail',
+    'password' => 'Mot de passe',
+    'language' => 'Langue',
+    'theme' => 'Theme',
```
Comment (samnela) [inline resources/lang/fr/fields.php]: Thème

**Comment by range-of-motion** [comment]
@KevinJaniky could you implement the suggestions from @samnela and update the translation files to mirror what's going on in master? Thanks!

**Comment by KevinJaniky** [comment]
@samnela  Thank's for this correction :) 
@range-of-motion  the repository is up to date and i added the suggestions from samnela .

**Pull request merged**